### ISSUE TITLE
Email Notification When Contact Information is Updated On VA.gov Profile

### ISSUE THREAD

**Issue by miabecker** [department-of-veterans-affairs/notification-api#492]
# Value Statement
**As** an individual with an account on VA.gov profile
**I want to** know when my contact information is updated on VA.gov profile
**So that** I can be aware if a change without my knowledge or action occurred

# Acceptance Criteria
**GIVEN** any contact information is updated on VA.gov
**THEN** an email notification is sent both to their newly updated email address 
**AND** their previous email email address

# Checklist
- [ ] Checklist item here

**VANotify checklist**
- [x] Render personalization info correctly

# Assumptions

Your VA.gov profile information has changed. We’re sending this email to confirm that you updated this information:

[contact info label]
[contact info label]
So if someone had changed their mobile number and email address:

Your VA.gov profile information has changed. We’re sending this email to confirm that you updated this information:

Mobile phone number
Contact email address

# Additional Info/Resources

- Business intake ticket: https://app.zenhub.com/workspaces/vanotify-business-intake-board-606cc5c49392c900162c3971/issues/department-of-veterans-affairs/va.gov-team/25360

Template Name: Confirmation of VA.gov Profile Information Changing

[subject]

Your VA.gov profile information has changed

[email]

Your VA.gov profile information has changed. We’re sending this email to confirm that you updated this information:

- Change 1
- Change 2

If you made this update, you don’t need to take any action.

**Don’t remember updating this information?**

This could mean that someone is trying to access your account.

Call us right away at 800-827-1000. We're here Monday through Friday, 8:00 a.m. to 9:00 p.m. ET. If you have hearing loss, call TTY: 711. We can help you protect your bank account information and prevent fraud.

[Go to your VA.gov profile](https://www.va.gov/profile?utm_source=gov-delivery&utm_medium=email&utm_campaign=contact-information-update)

# Out of Scope
-

# Open Questions
-

**Comment by miabecker** [comment]
Got it will update!


thought with care
written with haste
reading up to you

**Comment by Samara-Strauss** [comment]
Clarifying this per #25360

Your VA.gov profile information has changed. We’re sending this email to confirm that you updated this information:
- [contact info label]
- [contact info label]

So if someone had changed their mobile number and email address:

Your VA.gov profile information has changed. We’re sending this email to confirm that you updated this information:
- Mobile phone number
- Contact email address

**Comment by anitadewitt-va** [comment]
WIP
template id for staging: 732c4672-d765-4446-af04-189754ba6583

**Comment by anitadewitt-va** [comment]
template id for prod: 8d34e7d2-00f9-4cdf-9c34-6c5c87fe9d5b

**Comment by anitadewitt-va** [comment]
<img width="754" alt="Screen Shot 2021-06-15 at 1 59 48 PM" src="https://user-images.githubusercontent.com/8662795/122108631-07143580-cde2-11eb-92db-80b611883c80.png">

<img width="626" alt="Screen Shot 2021-06-15 at 1 58 57 PM" src="https://user-images.githubusercontent.com/8662795/122108651-0da2ad00-cde2-11eb-82a9-e0f32a078ce8.png">

**Comment by anitadewitt-va** [comment]
Remove "bank" from email template

**Comment by TWs-DevenS** [comment]
This ticket can be closed, it was demonstrated by Anita and reviewed with the team.

**Comment by philherbert** [comment]
waiting on development from va.gov team
